### PR TITLE
refactor faq with new orbit components

### LINKED ISSUE #275: Refactor static FAQs to use new orbit components
Refactor all components in `StaticFAQ` folder (FAQCategory, FAQArticle, FAQArticleDetail etc.) to use new orbit components - `Text`, `Heading` instead of deprecated `Typography`

### PR DESCRIPTION
**Pull request by jalbertsr**
Close #275 <br/><br/><br/><url>LiveURL: https://smartfaq-275-refactor-faq-with-orbit.surge.sh</url>

### CODE REVIEW

File: src/StaticFAQ/Breadcrumb.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ const Breadcrumb = ({ breadcrumb, isCurrent }: Props) => {
           .breadcrumb,
           .breadcrumb-icon {
             display: inline-block;
+            color: #7f91a8;
```
Comment (jalbertsr) [inline src/StaticFAQ/Breadcrumb.js]: this is only to change the colour for the `...`, it was inherited black before.

**Comment by dinodsaurus** [comment]
The thing that I noticed is that the buttons dont have the same `width` as earlier.
Button

**Comment by jalbertsr** [comment]
@dinodsaurus good catch! fixed.

**Comment by jalbertsr** [comment]
fixed, thanks!

**Pull request merged**